website/static/js/contribTreebeard.js

```diff
@@ -0,0 +1,114 @@
+'use strict';
+
+var $ = require('jquery');
+var m = require('mithril');
+var ko = require('knockout');
+var Treebeard = require('treebeard');
+var $osf = require('js/osfHelpers');
+var projectSettingsTreebeardBase = require('js/projectSettingsTreebeardBase');
+
+function expandOnLoad() {
+    var tb = this;  // jshint ignore: line
+    for (var i = 0; i < tb.treeData.children.length; i++) {
+        var parent = tb.treeData.children[i];
+        tb.updateFolder(null, parent);
+        expandChildren(tb, parent.children);
+    }
+}
+
+function expandChildren(tb, children) {
+    var openParent = false;
+    for (var i = 0; i < children.length; i++) {
+        var child = children[i];
+        var parent = children[i].parent();
+        if (child.children.length > 0) {
+            expandChildren(tb, child.children);
+        }
+    }
+    if (openParent) {
+        openAncestors(tb, children[0]);
+    }
+}
+
+function openAncestors (tb, item) {
+    var parent = item.parent();
+    if(parent && parent.id > 0) {
+        tb.updateFolder(null, parent);
+        openAncestors(tb, parent);
+    }
+}
+
+function NodesPrivacyTreebeard(data, nodesState, nodesOriginal) {
```

Comment:
Whereas `nodesPrivacySettingsTreebeard` *is* required (so we would expect this file to be required as well if it were actually in use):

```
$ ack nodesPrivacySettingsTreebeard
website/static/js/nodesPrivacy.js
13:var NodesPrivacyTreebeard = require('js/nodesPrivacySettingsTreebeard');
lines 1-2/2 (END) 
```